lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -228,6 +228,10 @@
       warn 'Skipping metadata validation; the metadata-json-lint gem was not found'
     end
   end
+
+  if File.exist?('REFERENCE.md')
+    Rake::Task['strings:validate:reference'].invoke
```

Comment:
The task may not be defined because there is graceful handling when the gem is not present:
https://github.com/puppetlabs/puppetlabs_spec_helper/blob/ac4945e2ccaa20a6868d29051d6e701aff73d3d7/lib/puppetlabs_spec_helper/rake_tasks.rb#L32-L36

I'd check if the task exists:
```suggestion
    if Rake::Task.task_defined?(:metadata_lint)
      Rake::Task['strings:validate:reference'].invoke
    else
      warn 'Skipping reference documentation validation; the puppet-strings gem was not found'
    end
```

Reply:
Makes sense! Updated to match the actual task we want to invoke.